src/main/java/duke/FileSaver.java

```diff
@@ -0,0 +1,128 @@
+/**
+ * This TaskList class handles the logic of adding and deleting tasks of Duke
+ * 
+ * @param file Task file to remember the tasks
+ * 
+ * @author WangYihe
+ * @author E0424695
+ */
+
+package duke.fileSaver;
+
+import duke.ui.Ui;
+import duke.exception.DukeException;
+import duke.task.*;
```

Comment:
Although all the three classes in task package are used, perhaps listing them out explicitly will comply with the coding standard better? 🤔